File: Python/tabel-perkalian.py
panjang = int(input("Masukkan panjang tabel perkalian: "))
lebar = int(input("Masukkan lebar tabel perkalian: "))

print("*\t", end="")
for i in range(1, panjang + 1):
    print(i, "\t", end="")
print()
for i in range(1, lebar + 1):
    print(i, "\t", end="")
    for j in range(1, panjang + 1):
        print(i * j, "\t", end="")
    print()

File: Golang/tabel-perkalian/tabel-perkalian.go
package main

import "fmt"

func main() {
	var panjang, lebar int

	fmt.Print("Masukkan panjang tabel perkalian: ")
	fmt.Scan(&panjang)
	fmt.Print("Masukkan lebar tabel perkalian: ")
	fmt.Scan(&lebar)

	fmt.Print("*\t")
	for i := 1; i <= panjang; i++ {
		fmt.Print(i, "\t")
	}
	fmt.Println()
	for i := 1; i <= lebar; i++ {
		fmt.Print(i, "\t")
		for j := 1; j <= panjang; j++ {
			fmt.Print(i*j, "\t")
		}
		fmt.Println()
	}
}
